_inc/connect-button.js

```diff
@@ -5,6 +5,7 @@ jQuery( document ).ready( function( $ ) {
 	var tosText = $( '.jp-connect-full__tos-blurb' );
 	connectButton.click( function( event ) {
 		event.preventDefault();
+		$('#jetpack-connection-cards').fadeOut(100);
```

Comment:
Couple of things here:

* Let's hear some design feedback on this, but I think `100` is just too fast. I'd suggest a value between 500 and 700 for this so it indeed looks like a fading out effect. Or we might as well just `.hide()` it if it's going to be that fast.
* Nit: There are some missing spaces.

```suggestion
		$( '#jetpack-connection-cards' ).fadeOut( 600 );
```